### ISSUE TITLE
create a html page (or your choice) to show the contributors along with their contributions.

### ISSUE THREAD

**Issue by t2013anurag** [hacktoberfest17/programming#463]
- [ ] Can be deployed in heroku or we can use github pages for this task.

- [ ] Design should be good and you can create your own sub-issues relating to the project.

**Comment by Username-anuj** [comment]
It'll be a static page, right??

**Comment by t2013anurag** [comment]
@Username-anuj - it should be dynamic, should make use of github-api

**Comment by jithurjacob** [comment]
@t2013anurag  I can work on this

**Comment by jithurjacob** [comment]
@t2013anurag  I've created a new "gh-pages" branch in my fork. But I'm not able to send PR as branch is not existing in this repo. Can you please create a "gh-pages" branch?

**Comment by jithurjacob** [comment]
Github is automatically serving the page under my domain. Once it is merged we could serve it as Github page from this repo. For now you could preview the page at https://jithurjacob.in/programming/

**Comment by Username-anuj** [comment]
@t2013anurag  Check this out : http://contributors.surge.sh/
It's dynamic. It picks contributors and their contributions from this repo.

**Comment by t2013anurag** [comment]
@Username-anuj and @jithurjacob - I like both the pages, you guys decide which should be selected as a final.

@jithurjacob - your page doesn't show all the contributors, can you look into that?

**Comment by t2013anurag** [comment]
@MadKat13 - any comments?

**Comment by t2013anurag** [comment]
@Username-anuj @jithurjacob I have created a branch.

**Comment by MadKat13** [comment]
@Username-anuj, Your solution doesn't show every contributor (look into pagination from github API). Your button doesn't work.

The style is very similar, but @jithurjacob solution is mobile friendly. 

Hmm the sub-issue task isn't solved yet.
I don't mind which one we take.

**Comment by t2013anurag** [comment]
@MadKat13 - agreed.
@Username-anuj @jithurjacob  - can we create a single script that uses card design of http://contributors.surge.sh/ and layout of https://jithurjacob.in/programming/ .

**Comment by t2013anurag** [comment]
Also since we are part of HacktoberFest should we show our contributors current status in a separate page (we can have a button or we can show in the card itself (9/4 pr completed or something.)), which fetches data from https://hacktoberfest.digitalocean.com/stats/t2013anurag

**Comment by t2013anurag** [comment]
<img width="467" alt="screen shot 2017-10-18 at 8 50 23 pm" src="https://user-images.githubusercontent.com/9719394/31726961-01835dd0-b446-11e7-9717-eb7c0b40b441.png">

**Comment by t2013anurag** [comment]
This design look good, similarly we can additional info as contributions  count to our HackertoberFest17/programming in the same design.

**Comment by MadKat13** [comment]
And since the most people make an algorithm problem (a new file) or a new folder, we could also link their contributions and write what they did.

**Comment by t2013anurag** [comment]
Yeah, that can be another feature.

**Comment by jithurjacob** [comment]
@t2013anurag  We can merge the gh-pages branch and work on making the UI changes.

**Comment by jithurjacob** [comment]
@t2013anurag  I'm unable to merge the branches getting the following response while comparing forks. Any ideas on how to proceed?

There isn’t anything to compare.
hacktoberfest17:gh-pages and jithurjacob:gh-pages are entirely different commit histories.

**Comment by t2013anurag** [comment]
@jithurjacob - Can you push the code for the ui as a separate PR, I will push it to the branch

**Comment by jithurjacob** [comment]
@t2013anurag  Please move the index.html to gh-pages branch. Also remove the other files gh-pages branch

**Comment by Username-anuj** [comment]
@t2013anurag  Thanks
@MadKat13 I have fixed the button. 
The page is responsive already.
I'll fix the github api pagination thingy. Thank you.

**Comment by Username-anuj** [comment]
It shows all the contributors now! Check : http://contributors.surge.sh/

**Comment by MadKat13** [comment]
@Username-anuj I like it, but if you want to merge it, fix your console.log statements ;)

**Comment by Username-anuj** [comment]
Oh yes! (:

**Comment by Username-anuj** [comment]
Done. I guess it is ready to be merged. What do you think? @MadKat13  @t2013anurag
